x/ecocredit/server/core/cancel.go

```diff
@@ -65,6 +65,7 @@ func (k Keeper) Cancel(ctx context.Context, req *core.MsgCancel) (*core.MsgCance
 			Address:        owner,
 			TradableAmount: userBalTradable.String(),
 			RetiredAmount:  userBalance.RetiredAmount,
+			EscrowedAmount: userBalance.EscrowedAmount,
```

Comment:
Good thing we are updating these tests... critical issue resolved here.

Reply:
good catch!

Reply:
> Good thing we are updating these tests... critical issue resolved here.

We have similar issue in send. We are not updating sender and recipient `EscrowedAmount` [send](https://github.com/regen-network/regen-ledger/blob/master/x/ecocredit/server/core/send.go#L123-L141).

Reply:
Yea, it looks like we do. Let's address this in a separate pull request and make sure we update tests.